codegenerator/cli/npm/envio/src/db/InternalTable.res

```diff
@@ -234,15 +370,19 @@ module DynamicContractRegistry = {
       mkField("registering_event_name", Text, ~fieldSchema=S.string),
       mkField("registering_event_src_address", Text, ~fieldSchema=Address.schema),
       mkField("contract_address", Text, ~fieldSchema=Address.schema),
-      mkField(
-        "contract_type",
-        Custom(Enums.ContractType.config.name),
-        ~fieldSchema=Enums.ContractType.config.schema,
-      ),
+      mkField("contract_name", Text, ~fieldSchema=S.string),
```

Comment:
This is a regression, but I think not a big one. If you find it critical, then I think there should be `envio_contracts` instead, with serial ids, and the `dynamic_contract_registry` only references the `contract_id` without `contract_name` and `chain_id` fields.